de1plus/dui.tcl

```diff
@@ -840,6 +848,7 @@ namespace eval ::dui {
 	
 			default.dscale.orient horizontal
 			default.dscale.foreground "#4e85f4"
+            default.dscale.activeforeground "#4e85f4"

```

Comment:
Whitespace error

Reply:
Pushed commit 199a088 to solve whitespace issues.